lib/date_time/date_time_core.c

```diff
@@ -32,6 +32,7 @@ static K_WORK_DELAYABLE_DEFINE(time_work, date_time_handler);
 
 static int64_t date_time_last_update_uptime;
 static date_time_evt_handler_t app_evt_handler;
+static int date_time_tz = DATE_TIME_TZ_INVALID;
```

Comment:
Add a comment that this is in quarters of an hour like in the AT command

Reply:
done